CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+## [2.20.2] - 2023-03-16
```

Comment:
why there are no changes?

Reply:
v2.20.1 failed to build (see follow-up https://github.com/giantswarm/cert-manager-app/pull/298). We could adapt the changelog afterwards. Didn't reword in _this_ PR yet because CI would add the new v2.20.2 entry on top.